frontend/public/components/monitoring/dashboards/index.tsx

```diff
@@ -349,16 +349,23 @@ const Card: React.FC<CardProps> = ({ panel }) => {
     );
   }
 
-  // Our grid has 12 columns, so don't allow colSpan over 12
-  const colSpan = Math.min(getPanelSpan(panel), 12);
-
-  // Don't allow very narrow panels at intermediate page widths
-  const colSpanMd = Math.max(colSpan, 3);
+  const panelSpan: number = getPanelSpan(panel);
+  // If panel.span is greater than 12, default colSpan to 12
+  const colSpan: number = panelSpan > 12 ? 12 : panelSpan;
+
+  let colSpanSize = 'max-1';
```

Comment:
Ideally this would be a `const` function that returns a value rather than a `let` that changes value.